evm/test/AggregatorProxy_test.js

```diff
@@ -79,12 +80,57 @@ contract('AggregatorProxy', () => {
     })
   })
 
+  describe('#updatedHeight', () => {
+    beforeEach(async () => {
+      const requestTx = await aggregator.requestRateUpdate()
+      const request = h.decodeRunRequest(requestTx.receipt.rawLogs[3])
+      await h.fulfillOracleRequest(oc1, request, response)
+      const height = await aggregator.updatedHeight.call()
+      assert.notEqual('0', height.toString())
+    })
+
+    it('pulls the height from the aggregator', async () => {
+      assertBigNum(
+        await aggregator.updatedHeight.call(),
+        await proxy.updatedHeight.call()
+      )
+    })
+
+    context('after being updated to another contract', () => {
+      beforeEach(async () => {
+        aggregator2 = await h.deploy(
+          'Aggregator.sol',
+          link.address,
+          basePayment,
+          1,
+          [oc1.address],
+          [jobId1]
+        )
+        await link.transfer(aggregator2.address, deposit)
+        const requestTx = await aggregator2.requestRateUpdate()
+        const request = h.decodeRunRequest(requestTx.receipt.rawLogs[3])
+        await h.fulfillOracleRequest(oc1, request, response2)
+        const height = await aggregator2.updatedHeight.call()
+        assert.notEqual('0', height.toString())
```

Comment:
Probably also useful to assert that it's a different value from the first `aggregator.updatedHeight()`.